File: beecrowd/javascript/beginner/1012-area.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8');
const lines = input.split('\n');

const [a, b, c] = lines
  .shift()
  .split(" ")
  .map(item => Number(item));
  
const pi = 3.14159;

const triangulo = (a * c) / 2;
const circulo = pi * c ** 2;
const trapezio = ((a + b) * c) / 2;
const quadrado = b ** 2;
const retangulo = a * b;

console.log(`TRIANGULO: ${triangulo.toFixed(3)}`);
console.log(`CIRCULO: ${circulo.toFixed(3)}`);
console.log(`TRAPEZIO: ${trapezio.toFixed(3)}`);
console.log(`QUADRADO: ${quadrado.toFixed(3)}`);
console.log(`RETANGULO: ${retangulo.toFixed(3)}`);

File: beecrowd/c++/beginner/1012-area.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    double a, b, c;
    double pi = 3.14159;
    
    cout << fixed << setprecision(3);
    
    cin >> a >> b >> c;
    
    cout << "TRIANGULO: " << a * c / 2 << endl;
    cout << "CIRCULO: " << pi * c * c << endl;
    cout << "TRAPEZIO: " << (a + b) * c / 2 << endl;
    cout << "QUADRADO: " << b * b << endl;
    cout << "RETANGULO: " << a * b << endl;
    
    return 0;
}
